src/ConnectionAutomaton.cpp

```diff
@@ -55,7 +55,7 @@ void ConnectionAutomaton::connect() {
         "",
         "",
         Payload());
-    onNextFrame(frame.serializeOut());
+    outputFrameOrEnqueue(frame.serializeOut());
```

Comment:
This connect method sticks out, especially since we probably want it to take a SetupFrameData with mimetype and lease modes eventually.